test/kixi/gwyn/data_test.clj

```diff
@@ -0,0 +1,23 @@
+(ns kixi.gwyn.data-test
+  (:require  [clojure.test :refer :all]
+             [clojure.java.io :as io]
+             [clojure.data.csv :as data-csv]))
+
+(defn open-csv [filename]
+  (with-open [in-file (io/reader filename)]
+    (vec (data-csv/read-csv in-file))))
+
+(def forbidden-fields ["Motive"
+                       "MainCause"
+                       "ParentMainCause"
+                       "MainCauseGroup"])
+
+(deftest test-for-forbidden-fields
+  (testing "csv's have forbidden fields present"
+    (let [files (map (fn [x] (.getPath x)) (file-seq (io/file "data/")))
+          csv-files (filter #(re-find #"\.csv$" %) files)]
```

Comment:
There's some inconsistency here between `fn` and `#` for anonymous functions. Another alternative is `partial`. Don't mind so long as it's consistent!